content/tutorial/04-advanced-sveltekit/04-advanced-loading/05-invalidate-all/README.md

```diff
@@ -0,0 +1,42 @@
+---
+title: invalidateAll
+---
+
+Finally, there's the nuclear option — `invalidateAll()`. This will indiscriminately re-run all `load` functions for the current page, regardless of what they depend on.
+
+Update `src/routes/[...timezone]/+page.svelte` from the previous exercise:
+
+```svelte
+/// file: src/routes/[...timezone]/+page.svelte
+<script>
+	import { onMount } from 'svelte';
+	import { +++invalidateAll+++ } from '$app/navigation';
+
+	export let data;
+
+	onMount(() => {
+		const interval = setInterval(() => {
+			+++invalidateAll();+++
```

Comment:
is it on purpose that you only did the `+++invalidateAll()+++` and not the `---invalidate(..)---` (same question applies to the other invalidate exercises)?

Reply:
it's just unnecessary — it's clear from context what you're supposed to remove. Having the red and the green butting against each other makes for visual chaos, and makes it harder to understand what you're meant to do IMHO